bin/readyapi_project.js

```diff
@@ -21,6 +21,8 @@ const xmlParserOptions = {
     parseTrueNumberOnly: false
 };
 
+const parser = new XMLParser(xmlParserOptions);
```

Comment:
Introduced because of breaking changes in fast-xml-parser 4.0.0 described here: https://github.com/NaturalIntelligence/fast-xml-parser/releases